testhelper/testhelper.py

```diff
@@ -71,7 +71,7 @@ def get_mount_parameter(test_info, share, combonum):
     """
     if (combonum > get_total_mount_parameter_combinations(test_info)):
         assert False, "Invalid combination number"
-    num_public = combonum / len(test_info["test_users"])
+    num_public = int(combonum / len(test_info["test_users"]))
```

Comment:
Is this a python3 specific change?

Reply:
It isn't Python 3 specific but was highlighted by the python3 interpreter. It should have been done when run under python2 as well.